src/app/dashboard/dashboard.component.css

```diff
@@ -0,0 +1,54 @@
+/* DashboardComponent's private CSS styles */
+
+h2 {
+  text-align: center;
+}
+
+.heroes-menu {
+  padding: 0;
+  margin: auto;
+  max-width: 1000px;
+
+  /* flexbox */
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-content: flex-start;
+  align-items: flex-start;
+}
+
+a {
+  background-color: #3f525c;
```

Comment:
It would be good to pull these hardcoded values for colour, font sizing, spacing out into a set of design token variables.

Reply:
@gavmck it'd probably help to explain what design token variables are here.